privacy-policy.md

```diff
@@ -0,0 +1,137 @@
+
+### Right to be forgotten
+
+You have the right to have us delete any personal data concerning you that we store.
+We have the obligation to erase your personal data without undue delay where one of the following grounds applies, as long as the processing is not necessary:
+
+-    The personal data are no longer necessary in relation to the purposes for which they were collected or otherwise processed
+-    The User withdraws consent to which the processing is based according to point (a) of Article 6(1) of the GDPR, or point (a) of Article 9(2) of the GDPR, and where there is no other legal ground for the processing
+-    The User objects to the processing pursuant to Article 21(1) of the GDPR and there are no overriding legitimate grounds for the processing, or the User objects to the processing pursuant to Article 21(2) of the GDPR
+-    The personal data have been unlawfully processed
+-    The personal data must be erased for compliance with a legal obligation in Union or Member State law to which we are subject
+-    The personal data have been collected in relation to the offer of information society services referred to in Article 8(1) of the GDPR
+
+### Right to data portability
+
+You have the right to ask for and receive the personal data we store about you in a structured, commonly used and machine-readable format. You also have the right to transfer this data to another controller without hindrance by the controller to whom the personal data have been provided, provided that the processing is based on a consent or on a contract to which the data subject is a party and that the processing is carried out by means of automated procedures.
+
+Furthermore, in exercising your right to data portability, you have the right to have personal data transmitted directly from one controller to another where technically feasible — and when doing so does not affect the rights and freedoms of others.
+
+### Right to withdraw consent
+
+You have the right to withdraw your consent regarding the use, processing or transmission of your data at any time in writing or by email to us.
+
+In the event of consent withdrawal, we will no longer process and immediately delete your stored data. This does not apply if we can prove compelling grounds for processing that are worthy of protection and which outweigh your interests, rights and freedoms or in case the processing serves to assert, exercise or defend legal claims. For example, we will continue to use data if it is still necessary for tax purposes.
+
+## Duration of Storage
+
+We store your data as long as necessary to organize Heart of Clojure, or as long as it is needed for legal or tax filing purposes.
+
+Generally we strive to delete all personal information within one month after the end of the conference.
```

Comment:
Conflicting with earlier sections that state hard deletion date as 1mo after conference.

Reply:
Updated to be unambiguous.